logic/peers.go

```diff
@@ -514,9 +352,32 @@ func GetPeerUpdateForHost(network string, host *models.Host) (models.HostPeerUpd
 		}
 	}
 
+	// run through delete nodes
+	if len(deletedNodes) > 0 {
```

Comment:
unnecessary check as for loop will be skipped if deletedNodes is empty.  The compiler will probably optimize out the check so not a big deal